source/IrcDotNet/StandardIrcClient.cs

```diff
@@ -144,14 +145,11 @@ public void Connect(string hostName, int port, bool useSsl, IrcRegistrationInfo
 
             if (registrationInfo == null)
                 throw new ArgumentNullException("registrationInfo");
-#if NETSTANDARD1_5
+
             var dnsTask = Dns.GetHostAddressesAsync(hostName);
             var addresses = dnsTask.Result;
 
             Connect(new IPEndPoint(addresses[0], port), useSsl, registrationInfo);
-#else
-            Connect(new DnsEndPoint(hostName, port), useSsl, registrationInfo);
-#endif
```

Comment:
I think we want to leave in this `#if` block, or at least adapt it to `TcpClient`.